src/BenchmarkDotNet/Extensions/ConfigurationExtensions.cs

```diff
@@ -22,11 +22,13 @@ public static string ToConfig(this Platform platform)
                     return "ARM64";
                 case Platform.S390x:
                     return "S390x";
+                case Platform.LoongArch64:
+                    return "LOONGARCH64";
```

Comment:
based on our observations, this is not needed. Let's remove it and merge the PR :shipit: 

```suggestion
```